Packs/qualys/Integrations/Qualysv2/Qualysv2.yml

```diff
@@ -30,6 +30,19 @@ configuration:
   required: false
   section: Connect
   advanced: true
+- display: Fetch Vulnerabilities Behavior
+  defaultvalue: Fetch by last modified date
+  name: fetch_vulnerabilities_behavior
+  options:
+  - Fetch by last modified date
```

Comment:
just want to clarify, `Fetch by last modified date` will be the default, right?

Reply:
Yes, this will maintain the existing behavior that customers are familiar with.